suite-native/state/src/configureStore.ts

```diff
@@ -1,11 +1,13 @@
-import { configureStore } from '@reduxjs/toolkit';
+import { combineReducers, configureStore, Store } from '@reduxjs/toolkit';
 
 import { appSettingsReducer } from '@suite-native/module-settings';
+import { appGraphReducer } from '@suite-native/graph';
 
-export const store = configureStore({
-    reducer: {
-        appSettings: appSettingsReducer,
-    },
+const rootReducer = combineReducers({
```

Comment:
Why these changes? What wrong with configure store?

Reply:
No particular reason. I was debugging and trying some stuff and decided to leave it like this as it's recommended in the docs but it's basically the same. Do you think it's unwanted? Nothing wrong with how it was as far as I'm concerned.

Reply:
@juriczech Yes, it is possible to do it in this way but I am sure there is nothing like "recommended" in the [documentation](https://redux-toolkit.js.org/usage/usage-with-typescript#getting-the-state-type). It's just about typing for app state. I think we are trying to use the new stuff where redux-toolkit (I mean configureStore) calls combineReducers automatically. It is not necessary to do it directly.

Reply:
@bouchja1 I just read [this](https://tppr.me/z2S7C), where it's mentioned that this might me more understandable for others but imo, I don't see why:D I just thought I'll leave it like this without giving it a second thought. I'll change it back to `configureStore`.

Reply:
Also, since this file's main purpose is to export store, shouldn't the name be `store.ts`?

Reply:
Fixed. @Nodonisko

Reply:
> Also, since this file's main purpose is to export store, shouldn't the name be `store.ts`?

well, here we can say it is somehow "[recommended](https://redux-toolkit.js.org/tutorials/quick-start#create-a-redux-store)" :D

Reply:
I don't see reason why we should change something that works well and it's according to docs. Also I think this is going to break thunks and possible some other stuff because configure store automatically adding some middlewares and enhancers. 

We can migrate it to manual way in case we will need to do something custom, but we don't need that now and probably won't ever need.

Reply:
Yeah, I agree. Fixed already.